Upstream-Status: Pending

Without this it fails to link against libtermcap causing various missing symbols 
issues.

RP - 8/10/08

Index: readline-5.2/configure.in
===================================================================
--- readline-5.2.orig/configure.in	2008-10-08 09:58:52.000000000 +0100
+++ readline-5.2/configure.in	2008-10-08 09:59:03.000000000 +0100
@@ -211,10 +211,10 @@
         AC_MSG_CHECKING(configuration for building shared libraries)
         eval `TERMCAP_LIB=$TERMCAP_LIB ${CONFIG_SHELL-/bin/sh} ${srcdir}/support/shobj-conf -C "${CC}" -c ${host_cpu} -o ${host_os} -v ${host_vendor}`
 
-#	case "$SHLIB_LIBS" in
-#	*curses*|*termcap*|*termlib*)	;;
-#	*)			SHLIB_LIBS="$SHLIB_LIBS $TERMCAP_LIB" ;;
-#	esac
+	case "$SHLIB_LIBS" in
+	*curses*|*termcap*|*termlib*)	;;
+	*)			SHLIB_LIBS="$SHLIB_LIBS $TERMCAP_LIB" ;;
+	esac
 	
         AC_SUBST(SHOBJ_CC)
         AC_SUBST(SHOBJ_CFLAGS)
